pkg/plumber/types.go

```diff
@@ -22,48 +22,19 @@ const (
 	BatchJob PlumberJobType = "batch"
 )
 
-// PlumberJobState specifies whether the job is running
-type PlumberJobState string
-
-// Various job states.
-const (
-	// TriggeredState means the job has been created but not yet scheduled.
-	TriggeredState PlumberJobState = "triggered"
-	// PendingState means the job is scheduled but not yet running.
-	PendingState PlumberJobState = "pending"
-	// SuccessState means the job completed without error (exit 0)
-	SuccessState PlumberJobState = "success"
-	// FailureState means the job completed with errors (exit non-zero)
-	FailureState PlumberJobState = "failure"
-	// AbortedState means prow killed the job early (new commit pushed, perhaps).
-	AbortedState PlumberJobState = "aborted"
-	// ErrorState means the job could not schedule (bad config, perhaps).
-	ErrorState PlumberJobState = "error"
-)
-
-// PlumberJob is used to request a pipeline to be created
-// its the lighthouse equivalent of a ProwJob
-// though its not a CRD directly; but a set of parameters used to actually create the
-// Tekton Pipeline CRDs
-//
-// By default we tend to turn Webhoks into tekton Pipeline CRDs
-type PlumberJob struct {
+// PlumberArguments contains the arguments to pass to the Plumber to create a Tekton Pipeline
+type PlumberArguments struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   PlumberJobSpec   `json:"spec,omitempty"`
-	Status PlumberJobStatus `json:"status,omitempty"`
+	Spec PlumberJobSpec `json:"spec,omitempty"`
```

Comment:
`PlumberJobSpec` doesn't sounds consistent with `PlumberArguments`, maybe we should rename it.

Reply:
on way...